analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/LocksShouldBeReleasedTest.cs

```diff
@@ -51,6 +51,18 @@ public class LocksShouldBeReleasedTest
         public void LocksShouldBeReleased_ReaderWriterLockSlim_CS() =>
             Verifier.VerifyAnalyzer(@"TestCases\SymbolicExecution\Sonar\LocksShouldBeReleased.ReaderWriterLockSlim.cs", new LocksShouldBeReleased());
 
+        [TestMethod]
+        [TestCategory("Rule")]
+        public void LocksShouldBeReleased_Mutex_CS() =>
```

Comment:
The rule has been renamed from `LocksShouldBeReleased` to `LockReleasedAllPaths`. Please apply the changes.

Reply:
This can be also fixed after the merge? Like the file locations?

Reply:
I`ll take care of it after the merge